usaspending_api/config/envs/default.py

```diff
@@ -77,6 +77,8 @@ def __new__(cls, *args, **kwargs):
     COMPONENT_NAME: str = "USAspending API"
     PROJECT_LOG_DIR: str = str(_SRC_ROOT_DIR / "logs")
 
+    TRACE_ENV: str = "local"
```

Comment:
This should follow a similar pattern to the `ENV_SPECIFIC_OVERRIDE` used by the `ENV_CODE` because we don't want "local" as a default

Reply:
Done.